pkg/operator/resources/cluster/cdiconfig.go

```diff
@@ -54,8 +54,23 @@ func createCDIConfigCRD() *extv1.CustomResourceDefinition {
 }
 
 func createBetaV1ConfigCRDSchema() *extv1.CustomResourceValidation {
-	// BetaV1 might diverge here from AlphaV1
-	return createAlphaV1ConfigCRDSchema()
+	// BetaV1 diverges from AlphaV1
+	config := createAlphaV1ConfigCRDSchema()
+	PatchBetaV1ConfigProps(config.OpenAPIV3Schema.Properties["spec"].Properties)
```

Comment:
Why updating in this way and not just changing `createAlphaV1ConfigCRDSchema()`?

Reply:
I was assuming adding it only to Beta is smarter than adding to both. I think @awels agreed?

Reply:
I was curious why using this patch mechanism?

Reply:
Afther the dedup we did here (Beta refering the Alpha func), I couldn't think of a cleaner way than a patch function to add the new Beta-only properties, but I'm open for any better suggestion :)

Reply:
I see no good reason to exclude this from alpha.  I don't think there is any benefit.

Reply:
We would have to also add it to alpha types.go and generate all associated files or the check to ensure that the generate manifest matches the code manifest will fail.

Reply:
sure

Reply:
done